api/agentpb/agent.proto

```diff
@@ -87,9 +87,8 @@ message QueryActionValue {
     int64 int64 = 3;
     uint64 uint64 = 4;
     double double = 5;
-    string str = 6;
```

Comment:
maybe it's better to keep this field, but change type to bytes to be able to differentiate byte and string while unmarshaling.

Reply:
There are several interconnected problems there.

1. protobuf does not allow invalid UTF-8 byte sequences in strings. Go allows them.

2. MySQL driver (or rather a combination of MySQL protocols, database/sql and the driver itself) returns `[]byte` we one could reasonably expect `string` or even other types like `int`. For example, there we use prepared statements to get `int`:
https://github.com/percona/pmm-agent/blob/32dba2c1f734aa6679bed4115d810a00910e467f/actions/mysql_query_select_action.go#L65-L81
https://github.com/percona/pmm-agent/blob/32dba2c1f734aa6679bed4115d810a00910e467f/actions/mysql_query_select_action_test.go#L37-L57
But here we get `[]byte` even with prepared statements:
https://github.com/percona/pmm-agent/blob/32dba2c1f734aa6679bed4115d810a00910e467f/actions/mysql_query_show_action.go#L65-L81
https://github.com/percona/pmm-agent/blob/32dba2c1f734aa6679bed4115d810a00910e467f/actions/mysql_query_show_action_test.go#L37-L58

3. We don't really know the encoding of data we receive from the database – our DSN does not specify one. We should specify it for query Actions now, and everywhere else later.

4. Finally, Starlark doesn't really care about strings encoding: https://github.com/google/starlark-go/blob/master/doc/spec.md#strings Our check scripts care (that's why point 3 above), but `string` variable in Starlark can be constructed from any Go `[]byte` sequence.

In the end, the difference between bytes and string only causes problems and does not add any benefits.

/cc @artemgavrilov